### PR TITLE
pre-commit update

### PR DESCRIPTION
**Pull request by goern**
- :sparkles: add python3.9 based pre-commit, opa, conftest
- :arrow_up: updated all to latest: opa, conftest, pre-commit, base image
- :pencil: update the readme to include pre-commit ubi9-py39

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,20 +17,21 @@ sudo skopeo copy containers-storage:localhost/thoth-python-36:latest docker://qu
...
+| thoth-mypy-ubi8        | deprecated | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-mypi/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-mypi)                         |
+| thoth-pytest-f31-py37  | deprecated | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-pytest-f31-py37/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-pytest-f31-py37)   |
+| thoth-black-ubi8       | deprecated | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-black/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-black)                       |
+| thoth-coala-ubi8       | deprecated | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-coala/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-coala)                       |
+| thoth-src              | deprecated | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-srcops/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-srcops)                     |
```
Comment (VannTen) [inline README.md:37]: That's for readme rendering with a table right ? Does the current works fine ?

File: thoth-precommit-py39/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
+FROM quay.io/thoth-station/s2i-thoth-ubi9-py39:v0.35.1
+
+ENV SUMMARY="Thoth's toolchain for running pre-commit hooks." \
+    DESCRIPTION="Thoth's toolchain for running pre-commit hooks."
+
+LABEL summary="$SUMMARY" \
+    description="$DESCRIPTION" \
+    io.k8s.description="$DESCRIPTION" \
+    io.k8s.display-name="Thoth Python 3.9-ubi9 Pre-Commit" \
+    io.openshift.tags="python,python38,pre-commit" \
```
Comment (VannTen) [inline thoth-precommit-py39/Dockerfile]:
```suggestion
    io.openshift.tags="python,python39,pre-commit" \
```
I suppose

**Review by VannTen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (goern) [reply README.md:37]: ja.

**Comment by goern** [comment]
ping

**Pull request merged**